test

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
```

Comment:
All of the dependencies for the tests should be vendored, so I don't think that would do anything.